tools/docker/nginx.apps.conf

```diff
@@ -1,14 +1,30 @@
 server {
-    listen       80;
-    listen  [::]:80;
-    server_name  localhost;
-    location / {
+        listen       80;
+        listen  [::]:80;
+        server_name  localhost;
+
         root   /usr/share/nginx/html;
-        index  index.html index.htm;
-        try_files $uri $uri/ /APP_NAME/index.html;
-    }
-    error_page   500 502 503 504  /50x.html;
-    location = /50x.html {
-        root   /usr/share/nginx/html;
-    }
+
+        server_tokens off;
+
+        location ~ /index.html|.*\.json$ {
```

Comment:
I think `/index.html` does not match anything in our case, the file is in `/APP_NAME/index.html` (`APP_NAME` is replaced at build time with e.g. `datahub`). See my other comment!

Reply:
Actually no, it works fine so forget what I said!